test/e2e/adminUI/tests/group002Home/uxTestHomeView.js

```diff
@@ -102,11 +102,11 @@ module.exports = {
 			.click('@nameListSubmenu')
 			.waitForListScreen();
 
-		browser.adminUIListScreen
-			.click('@singleItemDeleteIcon');
+		browser.adminUIListScreen.clickDeleteItemIcon([
+			{ row: 1, column: 1 }
```

Comment:
I'm still a little unsure about the `clickDeleteItemIcon` function. The delete icon is **always** in column 1 is it not?